test/common/index.js

```diff
@@ -30,7 +30,7 @@ const fs = require('fs');
 // Do not require 'os' until needed so that test-os-checked-fucnction can
 // monkey patch it. If 'os' is required here, that test will fail.
 const path = require('path');
-const util = require('util');
+const { inspect } = require('util');
```

Comment:
Diff would be smaller and honestly subsequent code easier to understand if we don't do this destructuring. `util.inspect(input)` is clear and not terribly long. `inspect(input)` is less obvious and happens hundreds of lines from here.

Reply:
(Above is a non-blocking comment.)

Reply:
I changed it back to `util` (the diff is bigger though, due to more lines needed elsewhere).